### PR TITLE
Add a shake functionality.

### PR DESCRIPTION
**Pull request by colin353**
- A port for shaking was created. A new javascript library
   was imported to handle shaking.
 - Right now, a shake triggers a debug message.

### CODE REVIEW

File: js/Update.elm [hunk 1]
```diff
@@ -8,13 +8,15 @@ import AnimationFrame
...
 import Server
+import Shake
 
 
 subscriptions : Model -> Sub Msg
 subscriptions model =
     Sub.batch
         [ Server.listen model ServerMsgReceived
         , AnimationFrame.times AnimationFrame
+        , Shake.shake (always Shake)
```
Comment (iteloo) [inline js/Update.elm]: Better put this inside the TradeStage case

File: web/index.html [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+    window.app = Elm.Main.embed(node, {
+        hostname: window.location.host
+    });
+
+    var shake = new Shake({
+        threshold: 15,
+        timeout: 1000
+    }).start();
+
+    setTimeout(() => {
```
Comment (iteloo) [inline web/index.html]: Keep trying after a timeout if adding event handler fails

File: web/shake.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
+/*
```
Comment (iteloo) [inline web/shake.js]: I'm sure this is impeccable since we didn't write it

File: js/Update.elm (hunk 1 above)
Comment (colin353) [reply js/Update.elm]: Done

File: web/shake.js (hunk 3 above)
Comment (colin353) [reply web/shake.js]: yes

**Pull request merged**